### ISSUE TITLE
Intermittent issues with PR previews

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/quansight-labs-site#257]
The preview build has not been consistent. I just recreated the project in CloudFare to also fix issues reported in #256 

@rgommers could you send a dummy PR against the blog? historically it has worked for me so need an extra hand to debug on a live PR

**Comment by mattip** [comment]
When I try to look at a preview I get 
```

There are no Identity Providers in your account

Please go to the Authentication Tab at https://dash.teams.cloudflare.com/settings/authentication 
to create an Identity Provider.

```

**Comment by rgommers** [comment]
They're not even intermittent, it just has never worked for me and for most other people without specific Cloudflare credentials. I'm inclined to just put the Netlify preview back, because this is a real pain point and overall it's just less work to go back to something that "just works". Cloudflare seems to want authentication, and all we want is a simple publicly available URL.

We're going to get rid of this whole infra within the next 6 months, so no need to plan for the long term too much.

@trallard @Carreau @tonyfast any objections to this?

**Comment by trallard** [comment]
Nah I can do the migration today. I checked in with Chris since I tried fixing the "need credentials" issue but it seems we are on a legacy account whatever that means.

**Comment by rgommers** [comment]
thanks!

**Comment by mattip** [comment]
Something is off. Does PR #278 need to rebase to get previews on netlify? [The error](https://app.netlify.com/sites/admiring-northcutt-a5be8c/deploys/618f6e155d96be000858fff6#L16) is
```
/opt/build-bin/run-build-functions.sh: line 205: /opt/buildhome/python3.7/bin/activate: No such file or directory
```

**Comment by IvanYashchuk** [comment]
`main` was merged into PR #278 and the preview now works there!

**Comment by rgommers** [comment]
It seems to be reliable across PRs now indeed:)
